server/queries/updateSurvey.test.js

```diff
@@ -67,7 +67,7 @@ it('Testing updateSurvey updates a document successfully.', async (done) => {
   //   dateClosed: 2,
   // });
   // console.log('updated doc: ', updatedDocument);
-  // expect(updatedDocument.status).toStrictEqual('published') &&
+  // expect(updatedDocument.status).toStrictEqual('active') &&
```

Comment:
same here, too many comments!!!

Reply:
not mine

Reply:
This test is commented out so we could get stuff on master while we had the issue with the dummy data. Now your branch is merged Toni we should be able to uncomment this today, but as João says it's not for him to do